app/static/src/app/components/options/SensitivityOptions.vue

```diff
@@ -18,6 +18,8 @@
         <sensitivity-param-values :batch-pars="batchPars"></sensitivity-param-values>
         <button class="btn btn-primary mb-4 float-end" @click="toggleEdit(true)">Edit</button>
       </div>
+      <hr/>
+      <sensitivity-plot-options></sensitivity-plot-options>
```

Comment:
I presume that this maps somehow to SensitivityPlotOptions, is that correct? Is this an automagic vue thing?